### PR TITLE
[Adminer] Issue with FastCGI (duplicate-headers)

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
This issue fixes local Problems (MAMP) I have with Adminer. MAMP uses fcgi, and fcgi throws an exception when multiple of the same HTTP Headers come. The issue is, that adminer sets HTTP-Headers using the header() function, and symfony sets them via the response object. Both of them set the Content-Type header and fcgi throws an error therefore. 

PS: It seems this fixes styling issues as well, but I am not sure, would be nice if somebody could confirm that.

### CODE REVIEW

File: pimcore/lib/Pimcore/Bundle/AdminBundle/Controller/Admin/External/AdminerController.php [hunk 1]
```diff
@@ -86,12 +87,10 @@ public function proxyAction(Request $request)
                         $content .= file_get_contents($this->adminerHome . 'designs/konya/adminer.css');
                         $content .= file_get_contents(PIMCORE_WEB_ROOT . '/pimcore/static6/css/adminer-modifications.css');
                     }
-
-                    $response->setContent($content);
                 }
             }
 
-            return $response;
+            return new Response($content, 200, array_merge($headers, $this->getAdminerSetHeaders()));
```
Comment (maff) [inline pimcore/lib/Pimcore/Bundle/AdminBundle/Controller/Admin/External/AdminerController.php]: Wouldn't `$response->headers->set($header, $value)` do the same as the `replace` argument defaults to true? Would be nicer than falling back to the array syntax IMHO. Maybe we could add an additional method which applies the headers from `getAdminerSetHeaders()` to a response object and use that method from both actions.

**Comment by brusch** [comment]
Finally :)

**Comment by dpfaffenbauer** [comment]
@maff I've updated the PR. There is now a mergeAdminerHeader function which takes the Response Object as parameter, and sets the headers.

@brusch Yeah, it's a must have feature for me, it so convenient to access the database directly within pimcore.

**Comment by maff** [comment]
Thanks! :)

**Pull request merged**